cli/src/commands/start.rs

```diff
@@ -182,6 +182,10 @@ pub async fn start(
                         enabled: false,
                         override_port: None,
                     },
+                    health_details: HealthOverrideSettings {
+                        enabled: true,
+                        override_port: None,
```

Comment:
you need to cross check wherever it does it now that port sent for the graphql vs health never matches else bail out before was not an issue before only 1 port but now 2 overrides have to handle that edge case